ops/storage.py

```diff
@@ -45,7 +47,9 @@ def _run(args: List[str], **kw: Any):
     cmd: Optional[str] = shutil.which(args[0])
     if cmd is None:
         raise FileNotFoundError(args[0])
-    return subprocess.run([cmd, *args[1:]], encoding='utf-8', **kw)
+    with tracer.start_as_current_span('ops.storage::subprocess.run') as span:
+        span.set_attribute('argv', args)
+        return subprocess.run([cmd, *args[1:]], encoding='utf-8', **kw)
```

Comment:
I feel like this is too ops-internal to be interesting to charmers -- and we're also telling them not to use Juju for storage, so until that changes, we shouldn't make it any better than it currently is.

Reply:
So... what do you reckon, is it better hide this entirely, even though we're calling an "external" binary?